setup-env.sh

```diff
@@ -358,12 +362,14 @@ fi
     *)
       save_var DEPLOYMENT_TYPE "${DEPLOYMENT_TYPE}" "$SAVE_ENV"
     ;;&
+    1|2|3|4|5)
+      save_var LICENSE_FILE "TRIAL" "$SAVE_ENV"
+      save_var DOCKER_HOSTNAME "sandbox-hdp" "$SAVE_ENV"
```

Comment:
I'm not sure the Docker_hostname is correct here?

Reply:
i've changed this